packages/financial-templates-lib/src/logger/PagerDutyTransportV2.ts

```diff
@@ -0,0 +1,49 @@
+// This transport enables winston logging to send messages to pager duty v2 api.
+import Transport from "winston-transport";
+import { event } from "@pagerduty/pdjs";
+
+type TransportOptions = ConstructorParameters<typeof Transport>[0];
+export type Severity = "critical" | "error" | "warning" | "info";
+
+interface Config {
+  integrationKey: string;
+  customServices?: { [key: string]: string };
+}
+
+export class PagerDutyTransportV2 extends Transport {
```

Comment:
would it not be better to replace the current pagerduty transport than have two?

Reply:
i dont think so, the apis are really different and it would force us to have to migrate everything at once, rather than maybe more controlled migration where we can pick and choose what we add, or add additional v2 calls while keeping the v1 calls in